internal/cmd/logout.go

```diff
@@ -63,25 +63,25 @@ $ infra logout --all --clear`,
 }
 
 func logoutOfServer(hostConfig *ClientHostConfig) (success bool) {
-	if !hostConfig.isLoggedIn() {
-		logging.Debugf("requested but not logged in to server [%s]", hostConfig.Host)
-		return false
-	}
```

Comment:
this never made sense; a user can be logged out due to expiry, but still need the settings unset.